### ISSUE TITLE
Errors on iOS are not sent to sentry.io when using with @sentry/vue

### ISSUE THREAD

**Issue by WIStudent** [getsentry/sentry-capacitor#91]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
@sentry/capacitor@0.4.0
@sentry/vue@6.11.0
@sentry/tracing@6.11.0
@capacitor/core@3.2.4
@capacitor/cli@3.2.4
@capacitor/ios@3.2.4

### Steps to Reproduce

I am using the following code to initialize sentry. This works on android and web:
```typescript
import Vue from 'vue';
import { Integrations } from '@sentry/tracing';
import * as Sentry from '@sentry/capacitor';
import * as SentryVue from "@sentry/vue";

console.warn('BEFORE INIT');

Sentry.init({
  enabled: true,
  Vue,
  dsn: "myDsn",
  environment: "environment",
  release: "release",
  ignoreErrors: ["CANCEL"],
  integrations: [new Integrations.BrowserTracing()],
  tracesSampleRate: 0.1
  }, SentryVue.init);

console.warn('AFTER INIT');

```
### Expected Result

Errors should be sent to sentry.io

### Actual Result

No error is sent to sentry.io on iOS. 

I think the native sentry plugin is not initialized correctly. `SentryCapacitor.initNativeSdk` never resolves and a `DataCloneError` gets thrown at some point. When trying to send something to sentry.io `SentryCapacitor.fetchNativeSdkInfo` rejects with `Called fetchSdkInfo without initializing cocoa SDK.`.

![Bildschirmfoto 2021-10-05 um 13 09 14](https://user-images.githubusercontent.com/2707930/136012192-2b974ad8-80b7-4c2b-9ef7-60970dc4897b.png)

**Comment by jennmueng** [comment]
@WIStudent If you remove the `Vue` option in the call to init does it work? I think it might be because we forgot to filter it out.

**Comment by bruno-garcia** [comment]
Good to hear you're not blocked. We should address this when adding proper Vue support.

**Comment by sawaca96** [comment]
@WIStudent which folder should I use for sourceMap ? `./dist` ? or `ios/App/App/public`?

**Comment by WIStudent** [comment]
@sawaca96 I didn't use sourceMaps with `@sentry/capacitor`.

**Comment by sawaca96** [comment]
@WIStudent Thank you for reply 👍

Android works, but iOS not working sourceMap 😂😂😂

**Comment by lucas-zimerman** [comment]
@sawaca96 the sourcemaps should be the same for both Android/iOS.
By any chance, are both Android/iOS versions of your app using the same release and dist?

**Comment by sawaca96** [comment]
> @sawaca96 the sourcemaps should be the same for both Android/iOS. By any chance, are both Android/iOS versions of your app using the same release and dist?
> 
> I usually upload the .dist folder, also make sure that sourcemaps are being generated when building your app on the dist folder with files named like "something.js.map" or "something.ts.map"

sure. android and ios has same sourceMap. and ./dist has *.js.map.

The same sourceMap and the same codebase return different information as below.
- ios
![image](https://user-images.githubusercontent.com/49309322/161650289-4518cdec-067f-4448-9c30-162cb1a843f0.png)
- android
![image](https://user-images.githubusercontent.com/49309322/161650344-1a4bfa64-9cdf-4b80-9031-f09cddf4c11a.png)

**Comment by lucas-zimerman** [comment]
@sawaca96 That's interesting. How did you capture this error? (for example you used sentry.capturemessage("Hi") or another call)
And how are you testing the iOS? debugging it on Xcode on a simulator or simply running the bundled ipa on a real device?

**Comment by sawaca96** [comment]
I use `throw new Error('Hi')`  and debuggin on Xcode simulator

**Comment by lucas-zimerman** [comment]
> I use `throw new Error('Hi')` and debuggin on Xcode simulator

Just to be sure, are you using Capacitor with Vue?

**Comment by sawaca96** [comment]
> Just to be sure, are you using Capacitor with Vue?

Yes.

**Comment by lucas-zimerman** [comment]
Thanks for clarifying, we'll investigate it 🙂

**Comment by sneko** [comment]
Hi all,

Just here to say I'm also facing the:
```
Called fetchSdkInfo without initializing cocoa SDK.
```

I'm on Vue v2 and Capacitor v2.

Here is the current init:
```ts
  Sentry.init(
    {
      Vue,
      dsn: process.env.xxxxxxxxx,
      tunnel: process.env.xxxxxxxxx,
      environment: process.env.xxxxxxxxx,,
      debug: false,
      release: process.env.xxxxxxxxxx,
      autoSessionTracking: true,
      integrations,
      // Native settings
      enableNative: true,
      enableNativeCrashHandling: true,
      enableNativeNagger: true,
    },
    SentryVue.init
  );
```

Any progress on this issue since then?

Thank you,

**Comment by lucas-zimerman** [comment]
This issue will be fixed on the next release :) 
Also @sawaca96 could you open a new issue with your specific problem in regard of your source mapping question?

**Comment by sawaca96** [comment]
@lucas-zimerman Thanks for reply ! 

before create new issue, I will try at new release 0.6.1

**Comment by WIStudent** [comment]
@lucas-zimerman @sneko After a long time I returned to my capacitor project today. I updated to `@sentry/capacitor@0.11.2` and `@sentry/vue@7.37.1`, and checked if my workaround is still needed. Unfortunately the fix in #170 did not work. I think the problem is that the fix filters the flied `vue`, but the field that actually needs to be filtered is `Vue` with a capital `V`.

I also found another field that needs to be filtered: `tracesSampler`
